hkcert2020-calm-down/release_files/chall.py

```diff
@@ -0,0 +1,72 @@
+import base64
+import binascii
+import hashlib
+import os
+import random
+from Crypto.PublicKey import RSA
+from Crypto.Util.number import bytes_to_long, long_to_bytes
+
+from secret import message
```

Comment:
The flag will be automatically added to the environment as an environment variable FLAG, so you can use `message = os.environ["FLAG"]` instead of using the secret import.

Reply:
not applicable because the message is different from the flag (for this challenge)